src/TabBarItem.js

```diff
@@ -201,7 +203,10 @@ export default function TabBarItem<T: Route>({
       onLongPress={onLongPress}
       style={tabContainerStyle}
     >
-      <View pointerEvents="none" style={[styles.item, itemStyle]}>
+      <View
+        pointerEvents="none"
+        style={[styles.item, itemStyle, tabBarItemStyle]}
```

Comment:
I forgot to pass `tabStyle` here. Can you change this to pass it instead? Then you won't need the new prop.

Reply:
oh great, I'll make this change and submit new pull request today ! thanks

Reply:
no worries, I just made the change and did a release

Reply:
oh great thanks